bundles/CoreBundle/EventListener/AssetSanitizationListener.php

```diff
@@ -47,11 +47,14 @@ public function sanitizeAsset(ElementEventInterface $e)
 
             if (isset($assetStream)) {
                 $streamMetaData = stream_get_meta_data($assetStream);
-                $mime = MimeTypes::getDefault()->guessMimeType($streamMetaData['uri']);
+                
```

Comment:
thank you for the PR, it would fix the specific case (which i was unable to reproduce with S3 adapter + Minio) but won't cover any other possible problem of `uri` field not returning a physical file, and it is actually skipping the sanitization (even though it should be fine for a copy/paste).

what about: 
```
if (file_exists($streamMetaData['uri'])){
    $mime = MimeTypes::getDefault()->guessMimeType($streamMetaData['uri']);}
else{
    $mime = mime_content_type($assetStream);
}

if ($mime === 'image/svg+xml') {
    $sanitizedData = $this->sanitizeSVG(stream_get_contents($assetStream));
    $element->setData($sanitizedData);
}
```
even though it was sort of "disrecommended" [here](https://github.com/pimcore/pimcore/pull/11386/commits/b5ea47670641ea14963dfc45c0ea937de56de742#r801659837) before i found out that could get it using stream_get_meta_data, would still be a quicker approach than write the stream in a temp file, read it with guessMimeType and delete the temp file. 
Let me know if this approach works as well

Reply:
Looks good to me, I have tested it in our repository and it seems to work!

Reply:
Maybe we should use `stream_is_local()` instead of `file_exists(...)`

Reply:
i just checkend and to my surprise, `var_dump(stream_is_local("php://temp"));` returns `bool(true)`

Reply:
but it should work for `stream_is_local($streamMetaData)` theoretically

Reply:
Ok 😅 But I'm fine with the current approach as well ... so no problem at all 😉